empyrical/utils.py

```diff
@@ -24,6 +24,16 @@
 import pandas as pd
 from pandas.tseries.offsets import BDay
 from pandas_datareader import data as web
```

Comment:
@twiecki @richafrank there is a known import error with `pandas_datareader`: it has been fixed upstream [here](https://github.com/pydata/pandas-datareader/pull/520), but has yet to be released in a new version. See [this StackOverflow thread](https://stackoverflow.com/questions/50394873/import-pandas-datareader-gives-importerror-cannot-import-name-is-list-like).

Wondering what we should do about this. Perhaps wrap it in a try-except, and raise a warning if the import fails? If we're deprecating `pandas-datareader` functionality anyways, it doesn't make sense to keep this line around if it raises an import error.